### ISSUE TITLE
Logits Vs Probs. Are they being mixed up?

### ISSUE THREAD

**Issue by demongolem** [CIA-Oceanix/GeoTrackNet#15]
I have run through the code (with actual data) and it seems to me that there is a mixing up of "probs" and "logits" in the code, but perhaps someone can correct me.

In runners.py we see `model(cur_inputs, rnn_sate, cur_mask, return_value = "probs")` which is current line 93 but then the only way the model is created is the `create_vrnn` function on line 250 of vrnn.py and then when this is called in runners.py in currently line 213 it is called with `vrnn.ConfitionalBernoulliDistribution` which is formed using the logits argument.  So when I run, I get a NoneType because we are really using logits while we are trying to evaluate using probs.

So first, I ran through in train mode and that completed successfully.  This problem that I have just reported arises then when I am trying to later run save_logprob mode.

**Comment by demongolem** [comment]
Ok I discovered the problem on my own.  We had made some slight adjustments I think to our codebase to get it to tf 2.0.0.  And tensorflow-probability is 0.8.0.  And save_logprob does work with those versions.  However, when running that version with save_logprob mode, we get a warning:

WARNING:tensorflow:From /somepath/tensorflow_probability/python/distributions/bernoulli.py:107: Bernoulli._probs deprecated behavior (form tensorflow_probability.python_distributions.bernoulli) is deprecated and will be removed after 2019-10-01.  Instructions for updating: "The `probs` property will return `None` when the distribution is parameterized with `probs=None`.  Use `props_parameter()` instead.

And, if a newer version of tensorflow-probability is used, this is not warning, but it is error.
